cmake/3rdParty/Platform/Windows/BuiltInPackages_windows.cmake

```diff
@@ -26,7 +26,7 @@ ly_associate_package(PACKAGE_NAME DirectXShaderCompilerDxc-1.6.2104-o3de-rev3-wi
 ly_associate_package(PACKAGE_NAME SPIRVCross-2021.04.29-rev1-windows                    TARGETS SPIRVCross                  PACKAGE_HASH 7d601ea9d625b1d509d38bd132a1f433d7e895b16adab76bac6103567a7a6817)
 ly_associate_package(PACKAGE_NAME tiff-4.2.0.15-rev3-windows                            TARGETS TIFF                        PACKAGE_HASH c6000a906e6d2a0816b652e93dfbeab41c9ed73cdd5a613acd53e553d0510b60)
 ly_associate_package(PACKAGE_NAME freetype-2.10.4.16-windows                            TARGETS freetype                    PACKAGE_HASH 9809255f1c59b07875097aa8d8c6c21c97c47a31fb35e30f2bb93188e99a85ff)
-ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.7.167-rev4-windows                     TARGETS AWSNativeSDK                PACKAGE_HASH a900e80f7259e43aed5c847afee2599ada37f29db70505481397675bcbb6c76c)
+ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.9.50-rev2-windows                      TARGETS AWSNativeSDK                PACKAGE_HASH 047de23fa57d33196666c22f45afc9c628bae354a6c39d774cbeee8054b2eb53)
```

Comment:
Curious: Why is this package rev2 instead of rev1?

Reply:
Never mind. Saw your changes in another PR: https://github.com/o3de/3p-package-source/pull/74/commits/176bee726e5e753ddeaa870f23f62479aafc1bcb